openhtf/core/measurements.py

```diff
@@ -198,7 +198,7 @@ def with_dimensions(self, *dimensions):
 
   def with_validator(self, validator):
     """Add a validator callback to this Measurement, chainable."""
-    if not callable(validator):
+    if not isinstance(validator, collections.Callable):
```

Comment:
not necessary? callable is in PY3.5 (as of PY3.2). PY3.3 is actually End-of-Life'd already (as of September 2017): https://www.python.org/dev/peps/pep-0398/#lifespan